platform/darwin/scripts/generate-style-code.js

```diff
@@ -2,12 +2,28 @@
 
 const fs = require('fs');
 const ejs = require('ejs');
-const spec = require('mapbox-gl-style-spec').latest;
 const colorParser = require('csscolorparser');
 
 const prefix = 'MGL';
 const suffix = 'StyleLayer';
 
+global.isObjectNotArray = function(object) {
+    return (typeof object === 'object' && !Array.isArray(object));
+}
+
+global.override = function(base, overrides) {
```

Comment:
Instead of rolling our own function, import `lodash` and use [`_.assign()`](https://lodash.com/docs/4.16.6#assign) or [`_.assignIn()`](https://lodash.com/docs/4.16.6#assignIn).